composer.lock

```diff
@@ -0,0 +1,1803 @@
+{
+    "_readme": [
```

Comment:
don't know why but I had to run `composer update` otherwise composer was complaining about the lock file not being up to date:
```
bash-5.1# php composer.phar install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.
Your lock file does not contain a compatible set of packages. Please run composer update.
```